classes/Kohana/Log.php

```diff
@@ -10,7 +10,7 @@
  * @copyright  (c) 2008-2012 Kohana Team
  * @license    http://kohanaframework.org/license
  */
-class Kohana_Log {
+class Kohana_Log extends Psr\Log\AbstractLogger implements Kohana_Log_Buffer {
 
 	// Log message levels - Windows users see PHP Bug #18090
 	const EMERGENCY = LOG_EMERG;    // 0
```

Comment:
I realise this has always been inconsistent (and always slightly will be for the syslog writer and/or if people are using syslog constants in their own code).

But it makes no sense that for the new logger on some (eg windows) systems `Kohana_Log::EMERGENCY == 1 == \Psr\Log\LogLevel::ALERT`.

In order of preference, we should:

* remove these constants (biggest bc break, best for us)
* make them point directly to the equivalent PSR LogLevel constant
* hardcode the integer values to match the keys of `$log_levels` below (unlikely to break anyone else's code unless they're a windows user relying specifically on `Kohana_Log::EMERGENCY === 1`)